docs/conf.py

```diff
@@ -416,3 +439,57 @@ def setup(app):
         universal_newlines=True,
     )
 )
+
+import shutil
+from pathlib import Path
+
+
+def copy_html_files(app, exception):
```

Comment:
copy paste from aiida-workgraph to move html files from the sphinx gallery folder to the sphinx source, so we can display the widgets